contrib/scripts/env-setup.py

```diff
@@ -20,11 +20,18 @@
 TORCHVISION_WHEEL_TMPL = 'torchvision-{whl_version}-cp{py_version}-cp{py_version}m-linux_x86_64.whl'
 
 
-def update_tpu_runtime(tpu_ip, version):
+def update_tpu_runtime(tpu_name, version):
   print(f'Updating TPU runtime to {version.tpu} ...')
-  url = 'http://{tpu_ip}:8475/requestversion/{tpu_version}'.format(
-      tpu_ip=tpu_ip, tpu_version=version.tpu)
-  print('Done updating TPU runtime: {}'.format(requests.post(url)))
+
+  try:
+    import cloud_tpu_client
+  except ImportError:
+    subprocess.call(['pip', 'install', 'cloud-tpu-client'])
```

Comment:
Can this be installed by default?

Reply:
Default as in on Colab and Kaggle runtimes? Colab pushed back previously on any additional packages so I doubt it.

Reply:
Covering my eyes to LGTM this 😄 
What about a maybe-sudo here?

Reply:
... never mind, that's `pip`